src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -259,10 +272,17 @@ class JibriSelenium(
             try {
                 HomePage(chromeDriver).visit(callUrlInfo.baseUrl)
 
+                var callStatsUsername = "jibri"
+                if (jibriSeleniumOptions.callStatsUsername.isNotEmpty()) {
+                    callStatsUsername = jibriSeleniumOptions.callStatsUsername
```

Comment:
Would it be best to still prefix this with something like `jibri-sip` ?

Reply:
I prefer letting the entity calling the start sip gateway service to decide how the callStatsUsername will be set. And that entity can decide to add sip-jibri prefix to the callstats username.

Reply:
Ok.  Maybe we want to call the `JibriSeleniumOptions` field `callStatsUsernameOverride`?

Reply:
Renamed it now, it does sound clearer this way